keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCacheTest.kt

```diff
@@ -170,8 +193,8 @@ object MemoryCloudDriverCacheTest {
   @Test
   fun `an invalid account, VPC id and region returns an empty set`() {
     every {
-      cloudDriver.listNetworks()
-    } returns mapOf("aws" to vpcs)
+      cloudDriver.listSubnets("aws")
+    } returns subnets
 
     expectThat(
       subject.availabilityZonesBy("test", "vpc-2", "external (vpc2)", "ew-west-1")
```

Comment:
Unclear to me how this test ever worked as it was stubbing the wrong CloudDriver call!